### PR TITLE
sortable amp ad implementation

### PR DESCRIPTION
**Pull request by Rooke**
This PR will be closed, I'm just using it for code review. The real PR will be push to ampproject/amphtml

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -105,6 +106,12 @@ export const adPreconnect = {
     'https://ads.rubiconproject.com',
     'https://optimized-by.rubiconproject.com',
   ],
+  sortable: [
+    'https://partner.googleadservices.com',
+    'https://securepubads.g.doubleclick.net',
+    'https://tpc.googlesyndication.com',
+    'https://tags-cnd.deployads.com',
```
Comment (bgahagan) [inline ads/_config.js]: tags-cdn

Comment (Rooke) [inline ads/_config.js]: wow, thanks.

Comment (bgahagan) [inline ads/_config.js]: Maybe tags-cdn should come first? Not sure if there is some priority, but tags-cdn should be the highest.

File: ads/sortable.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import {writeScript, validateDataExists} from '../src/3p';
+
+/**
+ * @param {!Window} global 
+ * @param {!Object} data
+ */
+export function sortable(global, data) {
+  validateDataExists(data, ['site', 'adname']);
+
+  const slot = global.document.querySelector('#c');
```
Comment (bgahagan) [inline ads/sortable.js]: `getElementById` might be a better option here

Comment (Rooke) [inline ads/sortable.js]: I was copying what doubleclick does here:
https://github.com/sortable/amphtml/blob/sortable-amp-ad/ads/doubleclick.js#L142

File: ads/sortable.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  const ad = global.document.createElement('div');
+  ad.className = 'ad-tag';
+  ad.setAttribute('data-ad-name', data.adname);
+  ad.setAttribute('data-ad-size', data.width + 'x' + data.height);
+  slot.appendChild(ad);
+  writeScript(global, 'https://tags-cdn.deployads.com/a/' + data.site + '.js');
```
Comment (bgahagan) [inline ads/sortable.js]: We're guaranteed to always be https right?  Or should we use a scheme-less uri (e.g. `//tags-cdn.deployads.com/a/`)

File: ads/sortable.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+-->
+
+# Sortable
+
+## Examples
+
+```html
+  <amp-ad width=728 height=90
+      type="sortable"
+      data-adname="728x90_amp"
```
Comment (bgahagan) [inline ads/sortable.md]: Why not `data-ad-name` to match our current tags?

File: ads/sortable.js (hunk 2 above)
Comment (Rooke) [inline ads/sortable.js]: Wow, not even close:
http://jsperf.com/getelementbyid-vs-queryselector

File: ads/sortable.js (hunk 3 above)
Comment (bgahagan) [inline ads/sortable.js]: Should this be `loadScript`?

File: ads/sortable.md (hunk 4 above)
Comment (Rooke) [inline ads/sortable.md]: Again, I'm following examples. Everyone seems to use the dot notation to access `data` properties, so I didn't want a '-'. Kind of lame, again.

File: ads/sortable.js (hunk 3 above)
Comment (Rooke) [inline ads/sortable.js]: Once again, just copying doubleclick, and others on the https://. Definitely should be (the asycronous) loadScript, good catch.

File: ads/sortable.md (hunk 4 above)
Comment (bgahagan) [inline ads/sortable.md]: what bout `data-name` then?

**Pull request closed without merging**